Realm/Tests/MigrationTests.mm

```diff
@@ -1603,6 +1603,10 @@ - (void)testConvertToEmbeddedAddingMoreLinks {
             XCTAssertNotNil(oldObject);
             XCTAssertNotNil(newObject);
             RLMObject *childObject = newObject[@"object"];
+            XCTAssertNotNil(childObject);
+            if (childObject == nil) {
```

Comment:
If the above assertion fails, is this even run?

Reply:
Yes. The rest of the test continues running even if an XCTest assertion fails.